manifests/f/flyingpie/windows-terminal-quake/1.6/flyingpie.windows-terminal-quake.installer.yaml

```
@@ -0,0 +1,15 @@
+PackageIdentifier: flyingpie.windows-terminal-quake
+PackageVersion: 1.6
+InstallerType: portable
+Commands:
+- windows-terminal-quake
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.WindowsTerminal
```

```suggestion
    - PackageIdentifier: Microsoft.UI.XAML.2.8
    - PackageIdentifier: Microsoft.WindowsTerminal
```
The install is still erroring, and I think it's having an issue with this dependency of the Windows Terminal.

---

@stephengillie Thank you for your feedback!

Could you explain what the UI.XAML.2.8 dependency changes or solves?